<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editscientist</class>
 <widget class="QDialog" name="Editscientist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>479</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>355</width>
    <height>479</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>355</width>
    <height>479</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="EditSciLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Edit scientist&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="nameLabelSci">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="sciNameLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="genderLabel">
     <property name="text">
      <string>Gender</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="femaleRadioPushButtonSci">
       <property name="text">
        <string>Female</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="maleRadioButton">
       <property name="text">
        <string>Male</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="birthYearLabel">
     <property name="text">
      <string>Birth year</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="sciBirthYearSpinBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <number>2015</number>
     </property>
     <property name="value">
      <number>2015</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Is this person alive?</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="yesAliveButton">
       <property name="text">
        <string>Yes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="noAliveButton">
       <property name="text">
        <string>No</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="deathYearLabel">
     <property name="text">
      <string>Death year</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="sciDeathYearSpinBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <number>2015</number>
     </property>
     <property name="value">
      <number>2015</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="confirmSciEditPushButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="backPushButtonEditSci">
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
